frontend/src/features/admin-form/common/components/PreviewFormBanner/PreviewFormBanner.tsx

```diff
@@ -127,12 +128,21 @@ export const PreviewFormBanner = ({
             </Button>
           )}
         </Flex>
-        <DuplicateFormModal
-          formId={formId}
-          isTemplate
-          isOpen={isModalOpen}
-          onClose={onModalClose}
-        />
+        {isTemplate ? (
+          <DuplicateFormTemplateModal
+            formId={formId}
+            isTemplate
+            isOpen={isModalOpen}
+            onClose={onModalClose}
+          />
+        ) : (
+          <DuplicateFormModal
+            formId={formId}
+            isTemplate
```

Comment:
we don't need to pass this here anymore right? and we can remove it as a prop from `DuplicateFormModal`